resize.js

```diff
@@ -103,7 +103,15 @@ function splitUrl(url) {
   return url.match(/^\/(.*)_(\d+)_(\d+)(_(\d+)x)?\.(.*)/);
 }
 
-function getImageParams(url) {
+function getImageParams(req) {
+  var url = req.url;
+  var queryParams = null;
+  var split = url.indexOf('?');
```

Comment:
You don't need an additional module. I just tried it locally and it does show up here (both with `console.log(req.query) directly and as a property of `console.log(req)`). Not sure what went wrong exactly, but for next time you can just use that instead.